server/init.go

```diff
@@ -8,16 +8,14 @@ import (
 
 // httpConn web server connection parameters
 type httpConn struct {
-	address    string
 	port       string
 	root       string
 	cascadeDir string
 }
 
 func main() {
 	httpConn := &httpConn{
-		address:    "localhost",
```

Comment:
What about if I'm trying to serve from a different address than localhost? Because you have eliminated the `Addr` part from the server handler you must serve the application on localhost. This is a bummer because if I'm trying to run the application from a WSL backed system (like in my case) you must specify the address you were given by the wsl host and that is definitely not localhost. I think changing the port only would suffice.

Reply:
I agree, but using `localhost` as the default hostname is bad, this will cause the program to bind to the same port already in use without error.

Did some refactoring, check it out.

Reply:
> This is a bummer because if I'm trying to run the application from a WSL backed system (like in my case) you must specify the address you were given by the wsl host and that is definitely not localhost. I think changing the port only would suffice.

Honestly, if that's the case, you should be passing the host as a flag, otherwise, you're modifying the literal string anyway, what's the difference?

Reply:
This looks much better, thank you. Sure a more elegant way would been to use some environment variables and use them both on the Makefile and in the program itself, but for the simple purpose to show some demos this was the simplest and easiest way.